backend/app/services/subscription/service.py

```
@@ -700,8 +722,12 @@ def calculate_next_execution_time(self, trigger_type, trigger_config):
         """Calculate next execution time."""
         return calculate_next_execution_time(trigger_type, trigger_config)
 
-    def _convert_to_subscription_in_db(self, subscription: Kind) -> SubscriptionInDB:
+    def _convert_to_subscription_in_db(
+        self, subscription: Kind, current_user_id: Optional[int] = None
```

Ruff (0.14.13)

Unused method argument: `current_user_id`

(ARG002)